LanguageFeatures/Wildcards/initializing_formals_A02_t02.dart

```diff
@@ -16,12 +16,12 @@ import '../../Utils/expect.dart';
 
 class C1 {
   int _, a;
-  C1(this.a) : _ = a;
+  C1(this.a) : _ = a; // OK, there is no name clash with the wildcard parameter.
```

Comment:
I'm not quite sure what this means. We don't have any wildcard parameters in this declaration.

I intended to suggest that we could have a case where there is another constructor of `C1` that _does_ have a wildcard parameter (unused), and we would still be able to initialize the instance variable named `_` because that wildcard parameter does not disrupt the resolution of `_` in the initializer element: `_ = a` always means "initialize the instance variable named `_` to the value of `a`".

```suggestion
  C1(this.a) : _ = a;
  C1.otherCase(this.a, String _) : _ = a;
```

Perhaps we have this already, it just occurred to me that we might not have a test that covers this situation (it is similar to "a wildcard doesn't shadow anything", but not quite the same).

Reply:
Ah... I misunderstood. Thank you, we had no such test. Updated

Reply:
Thanks!